var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -127,6 +127,10 @@ def with_or_without_hdf5(self, activated):
 
         return '--without-phdf5'
 
+    def setup_build_environment(self, env):
```

Comment:
I opened an issue report upstream that we can reference here:
```suggestion
    def setup_build_environment(self, env):
        # https://github.com/ornladios/ADIOS/issues/206
```